docs/ossm/istioctl/README.md

```diff
@@ -0,0 +1,81 @@
+[Return to OSSM Docs](../)
+
+# Installing the istioctl tool
+
+The `istioctl` tool is a configuration command line utility that allows service 
+operators to debug and diagnose Istio service mesh deployments.
+
+## Prerequisites
+
+* Use an `istioctl` version that is the same version as the Istio control plane 
```

Comment:
This is more like an install step rather than a prerequisite. What needs to happen before the user can see istioctl as a command line utility that they can download? Is it that the "Service Mesh Operator" needs to be installed? Does an instance of `Istio` need to exist? 

I don't see the option in the OpenShift Console (using 4.17) as shown in the animated GIF, and installing the OSSM 3 operator or  sail operator doesn't seem to be there yet, so I assume this isn't complete yet.

Reply:
> This is more like an install step rather than a prerequisite
Right, I'm going to move it out as an install step

> What needs to happen before the user can see istioctl as a command line utility that they can download?
This command line utility will appear only when the Service Mesh Operator will be installed (it is part of the deployed resources at installation time). So no need `Istio` resource to exist.

> I don't see the option in the OpenShift Console (using 4.17) as shown in the animated GIF, and installing the OSSM 3 operator or sail operator doesn't seem to be there yet, so I assume this isn't complete yet.
Yes it needs this PR to be merged and then the bundle to be published. I used and pointed to my own built container images to get this example GIF image.

Reply:
Doc will be added in a different PR